WireUI/Sources/WireIndividualToTeamMigrationUI/Views/Previews/ConfirmationPreview.swift

```diff
@@ -0,0 +1,33 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+import WireDesign
+import WireFoundation
+
+#Preview {
+    PageContainer(
+        content: {
+            ConfirmationView { _ in }
```

Comment:
I noticed that some texts are clipped when checking the dynamic type variants.
Can this be fixed?
![Screenshot 2024-12-04 at 13 22 44](https://github.com/user-attachments/assets/a683c255-4b95-4727-b14e-a0daf3ccdbcb)

Reply:
We’d have to wrap it all in a scroll view.

Reply:
done: https://github.com/wireapp/wire-ios/pull/2135/commits/1dea503e8a8ea8b9475d88e3e2afa358e815d3ff